thinkbox-deadline/azuredeploy-parameters.json

```diff
@@ -0,0 +1,31 @@
+{

```

Comment:
Please use the new schema for parameters. See https://github.com/Azure/azure-quickstart-templates/blob/master/100-starter-template-with-validation/azuredeploy.parameters.json